vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/SubmitWorkflowRequest.java

```diff
@@ -92,21 +92,20 @@ public String getWorkflowVersion() {
 
   @Override
   public int hashCode() {
-    return hashCodeIgnoreAttempt() * 31 + Integer.hashCode(attempt);
+    return hashCodeIgnoreAttemptAndConsumableResources() * 31 + Integer.hashCode(attempt);
```

Comment:
consumableResources probably should be included hashCode() as it is no longer in hashCodeIgnoreAttemptAndConsumableResources().

Reply:
hashCode is an override and called when creating several kinds of keys within and without vidarr. hashCodeIgnoreAttemptAndConsumableResources isn't, and is effectively (and should be, idk why it isn't) private. I am not sure why the code is formatted this way, but the intention is not to have two different hashes for the same object.